aat/src/resources/features/F-018/S-002.td.json

```diff
@@ -0,0 +1,28 @@
+{
+  "title": "should return a negative response when activity is missing",
+  "_guid_": "S-002",
+  "_extends_": "F-018_Test_Data_Base",
+  "specs": [
+    "an active profile in CCD",
+    "is to add an activity for the case created above",
+    "will miss activity information"
+  ],
+
+  "request": {
+
+    "body": {
+      "activity": ""
+    }
+  },
+    "expectedResponse": {
+      "responseCode": 422,
+      "headers": {
+        "_extends_": "Common_422_Response_Headers"
+      },
```

Comment:
This would better be 400 Bad Request. If not so, please open a Bug ticket and link here, but please also keep this scenario with its 422-based content.

Reply:
raised a ticket https://tools.hmcts.net/jira/browse/RDM-8986